### ISSUE TITLE
Why the recognition results of the trained recognition model cannot be replicated on the validation dataset

### ISSUE THREAD

**Issue by aftermath0703** [JaidedAI/EasyOCR#1189]
I have trained a recognition model, but there is a significant difference between the recognition results of the trained model on the validation set and the results during training. I believe that the incorrect positioning of the detector may be one of the reasons. I would like to ask which detector was used during training the recognition model

**Comment by abhinavrawat27** [comment]
@aftermath0703 have you found any solution. I am having similar problem, where the results during training was good but when I am doing OCR using the trained model, it is not working

**Comment by aftermath0703** [comment]
> @aftermath0703 have you found any solution. I am having similar problem, where the results during training was good but when I am doing OCR using the trained model, it is not working

Yeah, I found that recognize can directly replace readtext. The detect function is not referenced in the recognize function, and the entire image will be recognized directly.

The code is like `results = reader.recognize('xx.jpg')`

**Comment by abhinavrawat27** [comment]
Thanks mate, thanks for your help.
